src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -258,6 +258,53 @@ def test_aks_create_and_update_with_managed_nat_gateway_outbound(
             ],
         )
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(
+        random_name_length=17, name_prefix="clitest", location="eastus2euap"
+    )
+    def test_aks_create_with_block_and_update_to_none_outbound(
+        self, resource_group, resource_group_location
+    ):
+        aks_name = self.create_random_name("cliakstest", 16)
+        self.kwargs.update(
+            {
+                "resource_group": resource_group,
+                "name": aks_name,
+                "ssh_key_value": self.generate_ssh_keys(),
+            }
+        )
+
+        create_cmd = (
+            "aks create --resource-group={resource_group} --name={name} "
+            "--vm-set-type VirtualMachineScaleSets -c 1 "
+            "--aks-custom-headers AKSHTTPCustomFeatures=Microsoft.ContainerService/NetworkIsolatedClusterPreview "
+            "--outbound-type block "
+            "--bootstrap-artifact-source Cache "
+            "-k 1.30 "
+            "--enable-apiserver-vnet-integration "
```

Comment:
[Live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=103203892&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b&t=a26fe913-e2be-5062-b08e-d15f1acc7ea2&l=2737) failed with the following error
```
E                   azure.cli.core.azclierror.BadRequestError: (BadRequest) Feature Microsoft.ContainerService/EnableAPIServerVnetIntegrationPreview is not enabled. Please see https://aka.ms/aks/previews for how to enable features.
E                   Code: BadRequest
E                   Message: Feature Microsoft.ContainerService/EnableAPIServerVnetIntegrationPreview is not enabled. Please see https://aka.ms/aks/previews for how to enable features.
```

Reply:
[`live test build`](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=103209854&view=logs&j=b162b355-d59d-5864-ce0f-0a70f12dd28b&t=b162b355-d59d-5864-ce0f-0a70f12dd28b) succeeded in staging env.